### PR TITLE
chore(lockfile): Add package name into the lockfile

### PR DESCRIPTION
**Pull request by skateman**
`npm install` kept adding this line for me :thinking:

### CODE REVIEW

**Comment by bastilian** [comment]
@skateman that is odd... that shouldn't happen. What version of npm are you using?

**Comment by bastilian** [comment]
I'm using `7.16.0` and it doesn't seem to make that change then running `npm install`.

**Comment by skateman** [comment]
7.21.0

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "compliance-frontend",
```
Comment (akofink) [inline package-lock.json:8]: Isn't this the one that's on line 2?

**Review by akofink — APPROVED** [review]
I checked out the frontend-components package-lock.json, and it contains this as well. It doesn't seem to hurt anything to merge :+1:

**Pull request merged**